Changelog

```diff
@@ -6,7 +6,26 @@
 
 Next
 ====
-- Formally support Python 3.12.
+
+
+2.6.0
+=====
+release-date: !!! FIX_ME !!!
+:release-by: !!! FIX_ME !!!
```

Comment:
Hi @cclauss,

Please help me understand, are these dynamically generated?

Reply:
I am not a maintainer of this project so I do not know if there are tools for the dynamic generation of release notes.

I merely looked at previous entries in this file and at https://github.com/celery/django-celery-beat/releases and then manually built this similar content.  If maintainers have a better approach then this PR might go in the trash bin.

Thanks for your review (the checkmark in the upper right of this page).  I wish more visitors would take that approach because it is usually more convincing to project maintainers than scads of 👍 and 🚀 emojis.